src/Utility.php

```diff
@@ -26,4 +26,31 @@ public static function isExternalUrl($url) {
     return FALSE;
   }
 
+  /**
+   * Checks if an item is in a list which may use regular expressions.
+   *
+   * @param string $item
+   *   The item to check for.
+   * @param array $list
+   *   The list to check. Items in the list can use regex.
+   *
+   * @return bool
+   *   TRUE if the item is in the list and FALSE otherwise.
+   */
+  public static function inList($item, array $list) {
+    $found = FALSE;
+    if (is_array($list)) {
+      foreach (array_filter($list) as $needle) {
+        $pattern = preg_quote($needle, '/');
+        $pattern = str_replace('\*', '.*', $pattern);
+        preg_match('/^(' . $pattern . ')/', $item, $is_match);
+        if (!empty($is_match)) {
+          $found = TRUE;
+        }
+      }
```

Comment:
```suggestion
```

Reply:
Thanks! I'll change this in the code instead of accepting this only because this is in two parts with the check above.